import-service/serverless.ts

```diff
@@ -64,6 +94,13 @@ const serverlessConfiguration: Serverless = {
                   name: true
                 }
               }
+            },
+            authorizer: {
+              name: 'basicAuthorizer',
+              arn: '${cf:authorization-service-dev.basicAuthorizerArn}',
```

Comment:
'${cf:authorization-service-${self:provider.stage}.basicAuthorizerArn}' - why not?